scheduler/planner.go

```diff
@@ -249,9 +253,15 @@ func (unit *Unit) RankValue() int64 {
 		// of a bump, to avoid running older builds first.
 		avgLifeTime := timeInQueue / time.Duration(length)
 
+		mainlinePriority := int64(1)
 		if avgLifeTime < time.Duration(7*24)*time.Hour {
-			unit.cachedValue += priority * unit.distro.GetMainlineTimeInQueueFactor() * int64((7*24*time.Hour - avgLifeTime).Hours())
+			mainlinePriority = unit.distro.GetMainlineTimeInQueueFactor() * int64((7*24*time.Hour - avgLifeTime).Hours())
```

Comment:
we should do += since if GetMainlineTimeInQueueFactor is 0 (i think it is right now actually) we'll be setting priority to 0 too

Reply:
All the Get__Factor functions returns 1 if the factor is less than or equal to 0
Should I still make this change? I didn't know if adding the extra 1 would make a difference

Reply:
Ah okay that's fine then

Reply:
(Does look like tests need to be fixed though)

Reply:
it was off by 1 because the cachedValue would be 1 when it would normally be 0 so I went ahead and still had to make this change!